www/stammtisch_weiszer_stein.md

```diff
@@ -17,7 +17,6 @@ map_lon:      8.68857353925705
 map_zoom:     16
 gmaps_url:    https://maps.google.de/maps?q=weißer%20stein%20heidelberg
 ---
-Der Weiße Stein kombiniert Gerichte aus der ganzen Welt mit Zutaten
-aus der ganzen Region. Das rustikale Ambiente wird durch alte iMacs 
-(die mit farbigem Gehäuse und CRT) aufgehipstert. Im Sommer gibt es 
-auch einen kleinen Biergarten im Hinterhof.
+Der Weiße Stein bietet im Winter im Haus und im Sommer im Hinterhof
+eine gemütliche Location für den Stammtisch. Die Küche bietet sowohl
+gutbürgerliche, als auch Internationale Gerichte. Es gibt auch gute Burger!
```

Comment:
bei sowohl - als auch - Konjunktionen keine Kommata verwenden. Internationale in diesem Zusammenhang klein. Ich finde den Burger-Satz gut, aber so ein wenig alleinstehend. Da könnte man noch mal ran, aber soo wichtig isses mir auch nicht ;)